src/Symfony/Component/Console/Exception/FatalThrowableError.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Exception;
+
+class FatalThrowableError extends \ErrorException
```

Comment:
This, and the other one in `Security/Http` are copied from `Symfony\Component\Debug\Exception\FatalErrorException`, since there is no direct dependency.